client/lib/i18n-utils/node.js

```diff
@@ -1,13 +1,16 @@
 /** @format */
-
 /**
  * Internal dependencies
  */
-
 import config from 'config';
 
-module.exports = require( './utils.js' );
+export {
+	addLocaleToPath,
+	addLocaleToWpcomUrl,
+	getLanguage,
+	getLocaleFromPath,
+	isDefaultLocale,
+	removeLocaleFromPath,
+} from './utils';
```

Comment:
Huh. It looks we want the `export * from './utils';` syntax here, but I ran a quick test and it didn't run:

```
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){'use strict';import _Object$defineProperty from 'babel-runtime/core-js/object/define-property';import _Object$keys from 'babel-runtime/core-js/object/keys';Object.defineProperty(exports, "__esModule", { value: true });exports.getLocaleSlug = undefined;var _utils = require('./utils');
                                                                                                          ^^^^^^
    
    SyntaxError: Unexpected token import
```

I'm not sure what's going on there.

Reply:
I think that the problem is that Babel isn't handling this right. we should add a comment in the meantime.

what's happening is that it exports a `default` value instead of exporting all the named exports. once we stop compiling into CommonJS we can replace these with `export * from` statements.

Reply:
Noted in 71b41cece8